src/main/kotlin/io/snyk/plugin/services/SnykCliService.kt

```diff
@@ -113,7 +113,10 @@ class SnykCliService(val project: Project) {
         val additionalParameters = settings.getAdditionalParameters(project)
 
         if (additionalParameters != null && additionalParameters.isNotEmpty()) {
-            commands.add(additionalParameters)
+
+            val additionalParamsArray = splitParameters(additionalParameters)
```

Comment:
@vicmac thanks to pointing us on very important issue 👍! The only thing I could suggest is to simplify this part and just pass additionalParameters.split(" ") as parameter. Like 

```
if (additionalParameters != null && additionalParameters.isNotEmpty()) {
      commands.addAll(additionalParameters.split(" "))
}
```

Thanks!

Reply:
Thanks @aldanchenko. I resolved to put that into a separate function in case in the future you might want to:

1. Apply more complex logic to the parameter-splitting routine
2. Test the routine independently

If you think these are not a concern right now, I'll update the PR.

I was also thinking about updating the split expression to "` --`" since all params use the `--` prefix, I think that way it might be less prone to break and send a weird error messages if any extra spaces are incorrectly added to the additionalParams field.

Reply:
two cents from my side: CLI argument parsing is tricky (e.g. `--debug` can be `-d`). My suggestions would be:
1) split by whitespace
2) lets do the simplified version (without a separate function). We plan to add some improvements across all our Java integrations (this will resolve issue with args parsing). And without complex logic testing standard split function is not necessary.

Thanks both @vicmac and @aldanchenko.

Reply:
Makes sense, let's keep the whitespace char as delimiter